draft-foudil-securitytxt.md

```diff
@@ -421,7 +421,8 @@ sign-header      = <headers and line from section 7 of [RFC4880]>
 
 sign-footer      = <OpenPGP signature from section 7 of [RFC4880]>
 
-unsigned         = *line (canonical-field eol) (lang-field eol) *line
+unsigned         = *line [canonical-field eol] *line [lang-field eol] *line
+unsigned        /= *line [lang-field eol] *line [canonical-field eol] *line
 
```

Comment:
I think this is meant to be "=/ as per RFC 5234:
https://tools.ietf.org/html/rfc5234#section-3.3

The square brackets make sense

Reply:
I think we can only force for MUST not should

The contact field is in fact required, but I am not sure how to indicate that

Reply:
Nice catch! Oh and I'm going to move the `*line` between the Canonical and Preferred-Languages into the brackets for the Canonical field, that way we never have two `*line`s in a row.

For the 'Contact' directive... ah! Here's an obvious way to do it... to just change the definition of 'unsigned'

```abnf
unsigned =  *line [canonical-field eol *line] [lang-field eol *line] contact-field eol *line
unsigned =/ *line [canonical-field eol *line] contact-field eol *line [lang-field eol] *line
; { 4 more re-arrangements of those 3 things } 
```

This requires `3! = 6` alternative definitions. If we had another directive that requires this sort of special treatment, I can't seem to find a way to get it below `4! = 24` even with tricks like creating more ABNF rules. :slightly_frowning_face: 

This seems like it should be filed for further discussion therefore.